tensorflow/core/kernels/data/experimental/ignore_errors_dataset_op.cc

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 #include "tensorflow/core/framework/dataset.h"
 #include "tensorflow/core/framework/partial_tensor_shape.h"
 #include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/platform/logging.h"
```

Comment:
Internal tests are failing because this file does not directly depend on a module that provides this header file.

Please add the dependency "tensorflow/core/platform:logging" to this target.

Reply:
Thanks for your comment. I have added the dependency.